dubbo-admin-server/pkg/services/registry_service_sync.go

```diff
@@ -73,14 +78,14 @@ func (adminNotifyListener) Notify(event *registry.ServiceEvent) {
 	if strings.EqualFold(constant.EmptyProtocol, serviceUrl.Protocol) {
 		if services, ok := cache.InterfaceRegistryCache.Load(category); ok {
 			if services != nil {
```

Comment:
The application will panic if the type assertion fails. Please add an inspection for the type assertion like:

```go
if map, ok := services.(*sync.Map); !ok {
    // handle errors of type assertion
}
```

Reply:
PTAL, thx